cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/watch/ProjectWatcher.java

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
Let's move this and RunCommandExecutor classes to `io.ballerina.cli.utils` package

Reply:
Fixed with 9305aa7069635a900c85be9dcfd5d7a0452b032d